cognizant-studio-academy/User-Management
# FirstCommit

#### What's this PR do?
This PR contains Admin Login and Welcome Page Functionalities

#### Where should the reviewer start?
All Files in Client/ and Server/

#### How should this be manually tested?
Step 1 : With Valid Credentials , User clicks on Login button 
Step 2: After Successful login verify the Welcome page with Welcome message
Step 3: With Invalid Credentials , Check for the error message

#### What are the relevant tickets?
https://www.pivotaltracker.com/story/show/166152348
https://www.pivotaltracker.com/story/show/166152226
